CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## v0.3.2
+- Add missing symbol implementations for `downcase`, `upcase`, and `number-formatter` expressions [#58](https://github.com/mapbox/vtshaver/pull/58)
```

Comment:
It might be worth mentioning that `number-formatter` is very basic and doesn't use any of the config options passed in.